pkg/kubectl/resource_printer.go

```diff
@@ -240,8 +240,9 @@ func printPod(pod *api.Pod, w io.Writer) error {
 	if len(containers) > 0 {
 		firstContainer, containers = containers[0], containers[1:]
 	}
-	_, err := fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%s\t%s\n",
+	_, err := fmt.Fprintf(w, "%s/%s\t%s\t%s\t%s\t%s\t%s\n",
```

Comment:
Good enough, but I want to make a plea that we perhaps revisit this.  We're inventing ad-hoc syntax here, which makes it very hard to run this through simple tools like awk.  Why not just add another column?